http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -651,4 +722,99 @@ public String getTokenSignatureAlgorithm() {
         return tokenSignatureAlgorithm;
     }
 
+    public String getAuthenticationRequestFormat() {
+        return authenticationRequestFormat;
+    }
+
+    public void setAuthenticationRequestFormat(String requestObjectType ) {
+        this.authenticationRequestFormat = requestObjectType;
+    }
+
+    public String getRequestSignatureAlgorithm() {
+        return requestSignatureAlgorithm;
+    }
+
+    public void setRequestSignatureAlgorithm(String algorithm) {
+        this.requestSignatureAlgorithm = algorithm;
+    }
+
+    public String getRequestEncryptAlgorithm() {
+        return requestEncryptAlgorithm;
+    }
+
+    public void setRequestEncryptAlgorithm(String algorithm) {
+        this.requestEncryptAlgorithm = algorithm;
+    }
+
+    public String getRequestEncryptEncValue() {
+        return requestEncryptEncValue;
+    }
+
+    public void setRequestEncryptEncValue (String enc) {
+        this.requestEncryptEncValue = enc;
+    }
+
+    public String getRealmKey () {
```

Comment:
We shouldn't use "realm" here. We should rename this to better explain what this returns.

Reply:
I wasn't using these functions, so I removed anything related to realmKey that I added.